init_player_points.py

```diff
@@ -0,0 +1,37 @@
+""" Init (all)Players all status point
+"""
+import os
+
+from lib.sav_types.type import LevelSav
+from lib.sav_types.utilType import Null
+
+from lib.utils.level_util import loadSavJson, dumpSavJson
+from lib.utils.charactersave_util import getRawPlayers
+
+file_path = os.path.join(os.path.dirname(__file__), "Level.sav.json")
```

Comment:
Actually, I hope that someone will change this part to a normalized form, similar to a save tool task template.
(I will do it, not now, but in a week)